Document ASM Overview page (#37857)
DataDog/documentation: content/en/security/application_security/overview/threat_protection.md

---
title: Threat Protection
disable_toc: false
further_reading:
- link: "/security/application_security/overview/"
  tag: "Documentation"
  text: "App and API Protection Overview"
- link: "/security/application_security/overview/api_posture/"
  tag: "Documentation"
  text: "API Posture"
- link: "/security/application_security/overview/attack_summary/"
  tag: "Documentation"
  text: "Attack Summary"
---

> The **Threat Protection** page shows how well your services are instrumented and protected against attacks, the security signals they generate, the services most exposed to threats, and the trends in attack activity. Each section is described below.

Reviewer: Same directional-language issue as on the API Posture page.

```suggestion
The **Threat Protection** page shows how well your services are instrumented and protected against attacks, the security signals they generate, the services most exposed to threats, and the trends in attack activity. The following sections describe each area of the page.
```

## Ask Bits panel

> The **Ask Bits** panel is a contextual entry point to Bits AI for questions about your threat protection. It offers ready-made prompts such as **Top priorities**, **Summarize Last Week insights**, and **What I'm exposed to?** to help you begin an investigation without writing a query. The panel appears when Bits AI is enabled and you have the required access, and you can dismiss it.

Reviewer: The nearly-identical "Ask Bits panel" paragraph on `api_posture.md` uses "start an investigation" instead of "begin an investigation" — minor wording inconsistency between otherwise-matching boilerplate.

```suggestion
The **Ask Bits** panel is a contextual entry point to Bits AI for questions about your threat protection. It offers ready-made prompts such as **Top priorities**, **Summarize Last Week insights**, and **What I'm exposed to?** to help you start an investigation without writing a query. The panel appears when Bits AI is enabled and you have the required access, and you can dismiss it.
```

## App Instrumentation

The **App Instrumentation** section reports how broadly App and API Protection (AAP) is activated across your services. **Threat detection coverage** shows how many services are actively detecting threats in real time compared to your total services, and **Recommended services activated** tracks your progress in enabling AAP on at-risk services identified by known vulnerabilities and suspicious traffic. From this section you can protect additional services or view the services already protected.

## Attack Coverage

> The **Attack Coverage** section shows how well your services are protected against attack vectors. **Attack Tools** reports how many exposed services are protected from scanners, bots, and similar tooling through AAP monitoring and blocking, and **Exploit Prevention** reports how many services are protected from exploits through Runtime Application Self-Protection (RASP). The section also highlights services that need Threat Management enabled or a tracing library upgrade.

Reviewer: This introduces "Threat Management" undefined on a page titled "Threat Protection" — the similarity in names risks confusing readers about whether these are the same thing. Consider linking "Threat Management" to where it's defined, or adding a short clarifying phrase distinguishing it from "Threat Protection."

Author: I added a link to a section in a page where "Threat Management" is defined.
